pkg/virt-controller/watch/vmi_test.go

```diff
@@ -153,9 +154,10 @@ var _ = Describe("VirtualMachineInstance watcher", func() {
 		dataVolumeInformer, dataVolumeSource = testutils.NewFakeInformerFor(&cdiv1.DataVolume{})
 		recorder = record.NewFakeRecorder(100)
 
-		configMapInformer, _ = testutils.NewFakeInformerFor(&k8sv1.Pod{})
+		configMapInformer, _ = testutils.NewFakeInformerFor(&k8sv1.ConfigMap{})
+		pvcInformer, _ = testutils.NewFakeInformerFor(&k8sv1.PersistentVolumeClaim{})
 		controller = NewVMIController(
-			services.NewTemplateService("a", "b", "c", configMapInformer.GetStore()),
+			services.NewTemplateService("a", "b", "c", configMapInformer.GetStore(), pvcInformer.GetStore()),
```

Comment:
I have no idea on how to write a useful test for this over here...?
I will add a test in `template_test.go` for not existing PVC. Here we already have a test for adding the pod condition to the VMI when it can't be scheduled. I will also add a func test to `storage_test.go`, which tests if a non existing PVC will result in the correct VMI condition.